contents/handbook/people/team-structure/team-structure.md

```diff
@@ -48,32 +48,45 @@ We've organised the team into small teams that are multi-disciplinary. [You can
 
 Engineering is spread out into three small teams. Team Platform is a scope-specific team which focuses on deployments and ingestion (see [Team Platform](platform) for more details). App teams East and West work on the PostHog product in general and they're split based on timezones for optimal collaboration. We mostly hire full-stack engineers and each team has expertise across the stack of the entire product. To learn what each team is working on you can take a look at [feature ownership](/handbook/engineering/feature-ownership).
 
-#### [App East](app-east)
+#### [Product Analytics](product-analytics)
 - [Marius Andra](/handbook/company/team#marius-andra-software-engineer) (Team lead, Full Stack Engineer)
-- [Emanuele Capparelli](/handbook/company/team#ben-white-full-stack-engineer) (Growth Engineer)
 - [Michael Matloka](/handbook/company/team#michael-matloka-software-engineer) (Full Stack Engineer)
 - [Paul D'Ambra](/handbook/company/team#paul-dambra-software-engineer) (Full Stack Engineer)
 - _[Lottie Coxon](/handbook/company/team#lottie-coxon) (Graphic Designer) - pairs with Chris Clark_
-
-#### [App West](app-west)
-- [Eric Duong](/handbook/company/team#eric-duong-software-engineer) (Team lead, Full Stack Engineer)
-- [Neil Kakkar](/handbook/company/team#neil-kakkar-software-engineer) (Full Stack Engineer)
-- [Li Yi Yu](/handbook/company/team#li-yi-yu-full-stack-engineer) (Full Stack Engineer)
-- _[Chris Clark](/handbook/company/team#chris-clark) (Product Designer)_ - _Works across App West and App East_
+- _Hiring for a [full stack engineer](careers/full-stack-engineer)_
+- _Hiring for a [full stack engineer](careers/full-stack-engineer)_
+- Product: Supported by Annika Schmid
 
 #### [Session Recordings](session-recordings)
-- [Ben White](/handbook/company/team#ben-white-full-stack-engineer) (Full Stack Engineer)
+- [Ben White](/handbook/company/team#ben-white-full-stack-engineer) (Team lead, Full Stack Engineer)
 - [Alex Kim](/handbook/company/team#alex-kim-full-stack-engineer) (Full Stack Engineer)
-- [Rick Marron](/handbook/company/team#rick-marron-full-stack-engineer) (Full Stack Engineer)
+- [Emanuele Capparelli](/handbook/company/team#ben-white-full-stack-engineer) (Growth Engineer)
+- _Hiring for a [full stack engineer](careers/full-stack-engineer)_
+- Product: Supported by Annika Schmid
+
+#### [App West - Feature flags / experiments / cohorts / SQL](app-west)
```

Comment:
It seems odd to have just one team called "App <cardinal direction>" when the rest are  basically feature teams. Why not something like "Product Experimentation"?

Reply:
Agreed, I'll speak with Eric and come up with another name which encompasses the features that they own.